assets/training/model_management/src/azureml/model/mgmt/utils/exceptions.py

```diff
@@ -26,6 +26,7 @@ class ModelImportErrorStrings:
     GIT_CLONE_ERROR = "Failed to clone {uri}. Error: [{error}]"
     VM_NOT_SUFFICIENT_FOR_OPERATION = "VM not sufficient for {operation} operation. Details: [{details}]"
     CMD_EXECUTION_ERROR = "Error in executing command. Error: [{error}]"
+    MODEL_ALREADY_EXISTS = "Model with name {model_id} already exists in registry {registry}"
```

Comment:
Could we also specify a URL like: 

`https://ml.azure.com/registries/azureml/models/{model_name}/version/{model_version}`

Reply:
Done